microservices/CFDP/Gemfile.prod

```diff
@@ -0,0 +1,29 @@
+source ENV['RUBYGEMS_URL'] || "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 7.1.0'
```

Comment:
Recommend 7.2 to match our latest commits

Reply:
Updated to ~7.1 to support older and newer versions of cosmos